.github/workflows/firebase-hosting-merge.yml

```
@@ -0,0 +1,28 @@
+
+name: Deploy to Firebase Hosting on merge
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v1
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)